### PR TITLE
added names export for mercadopago module

### PR DESCRIPTION
**Pull request by lucmkz**
## 📝 Description

- Added named export to MercadoPago configuration module
- Modified integration example in readme
- Adjusted PR template

### ✅ Checklist:

- [x] I followed the instructions indicated in [Contribution Guidelines](CONTRIBUTING.md)
- [x] Added the necessary tests for the code or functionality that I'm working on
- [x] I ran the tests and made sure they work
- [x] My branch coverage is at least 80%
- [x] I verified the style of my code matches the one in the project

## 🧰 How to reproduce

- Step by step of how to test, specially for bugs.
- Links of external docs.

## 📸 Screenshots

Not Apply

## 📄 References

Not Apply

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { MercadoPagoConfig } from './mercadoPagoConfig';
 export default MercadoPagoConfig;
+export { MercadoPagoConfig };
```
Comment (rhames07) [inline src/index.ts]: É necessário fazer essas duas exportações da mesma classe?

Comment (AdaltonLeite) [reply src/index.ts]: Nesse caso @rhames07  acredito que foi exportado assim para ser utilizado de diferentes formas no momento do import

**CommonJS**
```
var {MercadoPago} = require('...')
```

e para o EsModule: 

```
import MercadoPago from '...' 
```

usando somente o `export default MercadoPago` quando foi importado usando **commonJS**, tivemos que usar o `MercadoPago.default`

File: src/index.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import { MercadoPagoConfig } from './mercadoPagoConfig';
-export default MercadoPagoConfig;
+import { MercadoPagoConfig as MercadoPago } from './mercadoPagoConfig';
```
Comment (romerosilva-meli) [inline src/index.ts:1]: Why do we need to rename `MercadoPagoConfig`?

**Review by romerosilva-meli — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (lucmkz) [reply src/index.ts:1]: To maintain compatibility with those who already have the integration

File: src/index.ts (hunk 1 above)
Comment (lucmkz) [reply src/index.ts]: sim, a ideia desse PR é exatamente ter duas formas de exportação do modulo

**Review by romerosilva-meli — APPROVED** [review]

**Pull request merged**